test/test_http_instrumentation.py

```diff
@@ -536,59 +528,59 @@
 # are only loaded from secure contexts.
 HTTP_SERVICE_WORKER_REQUESTS = {
```

Comment:
We can probably remove the comment above this line that references `localtest.me` now

Reply:
Done